### PR TITLE
Implement russian language into the game

### PR DESCRIPTION
**Pull request by BurAndBY**
[TranslationsRU.zip](https://github.com/GeoGuess/Geoguess/files/5548568/TranslationsRU.zip)
[Header.zip](https://github.com/GeoGuess/Geoguess/files/5548576/Header.zip)
[index.js.zip](https://github.com/GeoGuess/Geoguess/files/5548775/index.js.zip)

### CODE REVIEW

**Comment by BurAndBY** [comment]
I don't sure this is working lol. ### **_My programming skills are 0_**

File: src/lang/TranslationsRU.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            },
+            aboutDescriptions: {
+                1: 'Geoguess это бесплатная и ленивая игра без рекламы. \
+                Игроки соревнуются, насколько близко игрок может угадывать случайные места за пять раундов. \
+                Вы можете поделиться счетом с другими людьми через социальные сети, такие как Facebook или Twitter. \
+                Вы можете играть в многопользовательскую игру со своими друзьями до пяти друзей. \
+                Первый игрок создает комнату и определяет ее размер. Другие игроки вводят то же имя комнаты, что и первый созданный игрок, и игра начнется. ',
+                2: 'Эта игра была разветвлена ​​от GeoGuess Master ',
+            },
+            customMap: 'CUSTOMS MAP',
```
Comment (BilelJegham) [inline src/lang/TranslationsRU.js:19]: @BurAndBY Can you translate this string ? :)

File: src/lang/TranslationsRU.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        },
+        CardRoomDifficulty: {
+            title: 'Установите уровень сложности.',
+            easy: 'Легкий (мир)',
+            medium: 'Средний (страна)',
+            hard: 'Сложный (город)',
+        },
+        DialogSummary: {
+            summaryMsgSingle:
+                'Вы попали <strong>{distance}</strong> km отсюда! Your score is <strong>{points}</strong> points!',
```
Comment (BilelJegham) [inline src/lang/TranslationsRU.js:128]:
```suggestion
               Your score is <strong>{points}</strong> points!',
```
Same ;)

File: src/lang/TranslationsRU.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            summaryMsgMulti:
+                '<strong>{playerName}</strong> количество очков <strong>{points}</strong> ! (<strong>{distance}</strong> kmaway)',
```
Comment (BilelJegham) [inline src/lang/TranslationsRU.js:130]: `kmaway` last ;)

**Review by BilelJegham — CHANGES_REQUESTED** [review]
Thanks @BurAndBY, some strings aren't translated. Can you check it ?

File: src/lang/TranslationsRU.js (hunk 1 above)
Comment (BurAndBY) [reply src/lang/TranslationsRU.js:19]: Oh yeah thanks

File: src/lang/TranslationsRU.js (hunk 3 above)
Comment (BurAndBY) [reply src/lang/TranslationsRU.js:130]: https://pastebin.com/NdYTL1g4 edited one. GitHub don't wanna upload the file.

**Comment by BurAndBY** [comment]
https://pastebin.com/NdYTL1g4
[TranslationsRU.zip](https://github.com/GeoGuess/Geoguess/files/5552215/TranslationsRU.zip)

**Comment by BurAndBY** [comment]
> Thanks @BurAndBY, some strings aren't translated. Can you check it?

FIXED

**Review by BilelJegham — APPROVED** [review]

**Comment by BilelJegham** [comment]
@all-contributors please add @BurAndBY for translation

**Pull request merged**